### PR TITLE
circle: Fix build and upload

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,21 +21,21 @@ jobs:
...
           mkdir /tmp/bottles
           cd /tmp/bottles
-          umask 022;
-          PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH";
+          umask 022
+          PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH"
           brew install patchelf
           brew tap linuxbrew/xorg
-          brew test-bot --tap=homebrew/core --bintray-org=linuxbrew --git-name=LinuxbrewTestBot --git-email=testbot@linuxbrew.sh --keep-old
+          brew test-bot --tap=linuxbrew/extra --bintray-org=linuxbrew --git-name=LinuxbrewTestBot --git-email=testbot@linuxbrew.sh
```
Comment (sjackman) [inline .circleci/config.yml]: Remove `--tap=linuxbrew/extra`

File: .circleci/config.yml [hunk 2]
```diff
@@ -21,21 +21,21 @@ jobs:
       - run: git config --global user.name LinuxbrewTestBot
       - run: git config --global user.email testbot@linuxbrew.sh
       - run: chmod 0644 Formula/*.rb
-      - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew
-      - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/$CIRCLE_PROJECT_REPONAME
+      - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
+      - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew/$CIRCLE_PROJECT_REPONAME
```
Comment (sjackman) [inline .circleci/config.yml]: Change both `linuxbrew` to `$CIRCLE_PROJECT_USERNAME`

**Review by sjackman — CHANGES_REQUESTED** [review]

**Comment by sjackman** [comment]
I'd like to remove
```yaml
       HOMEBREW_VERBOSE: 1
       HOMEBREW_VERBOSE_USING_DOTS: 1
```
that can be done in a separate PR or this one if you like.

**Review by sjackman — APPROVED** [review]

**Comment by iMichka** [comment]
Let's do that in a separate PR.

Why do you want to remove it? We have it also in xorg.

**Comment by sjackman** [comment]
It creates unnecessary noise in the log. For example, every file in the source tarball is listed. Adding it was a hack to avoid CircleCI ten-minute-without-output timeouts. There's a better way to avoid timeouts by modifying the YAML if it's ever needed.

**Pull request closed without merging**